models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1625,7 +1625,8 @@ FROM
     ('val-valeria', 'ethereum', 'VAL', 0x011e128ec62840186f4a07e85e3ace28858c5606, 18),
     ('propc-propchain', 'ethereum', 'PROPC', 0x9ff58067bd8d239000010c154c6983a325df138e, 18),
     ('beam-beam-eth', 'ethereum', 'BEAM', 0x62d0a8458ed7719fdaf978fe5929c6d342b0bfce, 18),
-    ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18)
+    ('sdl-stakelink', 'ethereum', 'SDL', 0xa95c5ebb86e0de73b4fb8c47a45b792cfea28c23, 18),
+    ('ton-tokamak-network', 'ethereum', 'TON', 0x2be5e8c109e2197D077D13A82dAead6a9b3433C5, 18)
```

Comment:
API: https://api.coinpaprika.com/v1/coins/ton-tokamak-network

Reply:
Thanks~ @Hosuke

Reply:
Sorry @kooroot, this PR is not merged yet. You may need to reopen this PR.